src/theory/bv/bv_solver_bitblast.cpp

```diff
@@ -346,27 +346,24 @@ bool BVSolverBitblast::collectModelValues(TheoryModel* m,
 
 void BVSolverBitblast::initSatSolver()
 {
-  switch (options().bv.bvSatSolver)
-  {
-    case options::BvSatSolverMode::CRYPTOMINISAT:
-      d_satSolver.reset(prop::SatSolverFactory::createCryptoMinisat(
-          statisticsRegistry(),
-          d_env.getResourceManager(),
-          "theory::bv::BVSolverBitblast::"));
-      break;
-    default:
```

Comment:
This code ignores the option for kissat, and generates a cadical instance instead. Is this by design?